lib/kb_Metrics/metricsdb_controller.py

```diff
@@ -30,9 +30,46 @@ def _config_str_to_list(self, list_str):
 
         return user_list
 
+    def _parse_app_id(self, exec_task):
```

Comment:
I mentioned this func in my previous review but you should really go over your code and make sure all private funcs are named like this. I won't put a comment on every funcs.

Reply:
This one is done now.